type_classes.md

```diff
@@ -183,20 +319,26 @@ but is expected to have type
 ```
 The theorem `defNatEq0` type checks because the type checker can unfold `(default : Nat)` and reduce it to `0`. This is not the case in the theorem `arbitraryNatEq0` because `arbitrary` is an opaque constant.
 
+BUGBUG: is "type checks" the right phrase? It doesn't seem to be used in propositions_and_proofs.html.
```

Comment:
Can be proved does certainly sound better from a mathematician's POV here. I would guess that from a type theorist's point of view "type checks" is basically equivalent in this context.